pages/api/calendar.js

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from 'luxon';
+
+import { getTalks } from '../../lib/airtable';
+
+const currentDate = DateTime.local();
+
+const removeOldTalks = talk => new Date(talk.date) > new Date();
+
+const futureTalks = month => ({
+    date: currentDate.set({ month }).toISO(),
+});
+
+const completeNextYearTalks = sizeCurrentMonth =>
+    Array.from(
+        {
+            length: 12 - sizeCurrentMonth,
+        },
+        (_, i) => futureTalks((i + new Date().getMonth() + 1 + sizeCurrentMonth) % 12),
+    );
+
+export default async (req, res) => {
+    try {
+        const talks = await getTalks();
+
+        const futureTalks = talks.filter(removeOldTalks);
+        const calendar = [...futureTalks, ...completeNextYearTalks(futureTalks.length)];
```

Comment:
This works fine unless there is a hole in the talks planning, like, what if there is a talk in february, april, but not in march?

Reply:
Two solution:
- change my code for this case
- delete this code and add `pending` talk in airtable (easier solution, create 20-30-x lignes pending and keep the first 12)

Reply:
This is why I had the horrible twelve nulls array :D
Maybe change this step in a `reduce` to add the missing month until we got 12 of them ?

Reply:
Do it in another PR